src/pages/images/ImageSelector.tsx

```diff
@@ -272,6 +275,7 @@ const ImageSelector: FC<Props> = ({ onSelect, onClose }) => {
                 type="button"
                 dense
                 className="u-no-margin--bottom"
+                appearance={item.cached ? "positive" : "default"}
```

Comment:
Should we also have the green one for local images?

Reply:
We can instead sort the cached images to the top of the list and keep the current behaviour of the "postive" and default button appearance.

Reply:
implemented the sorting :+1: